h/formatters/annotation_flag.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from zope.interface import implementer
+
+from h import models
+from h.formatters.interfaces import IAnnotationFormatter
+
+
+@implementer(IAnnotationFormatter)
+class AnnotationFlagFormatter(object):
```

Comment:
I’d find a top-level docstring really useful here, particularly coming back to this in the future. I had to look through the source code to confirm that `AnnotationFlagFormatter` means “formatter for annotations that includes information about flags”.